doc/classes/PortableCompressedTexture2D.xml

```diff
@@ -19,10 +19,12 @@
 			<param index="1" name="compression_mode" type="int" enum="PortableCompressedTexture2D.CompressionMode" />
 			<param index="2" name="normal_map" type="bool" default="false" />
 			<param index="3" name="lossy_quality" type="float" default="0.8" />
+			<param index="4" name="near_lossless" type="bool" default="false" />
 			<description>
 				Initializes the compressed texture from a base image. The compression mode must be provided.
 				If this image will be used as a normal map, the "normal map" flag is recommended, to ensure optimum quality.
 				If lossy compression is requested, the quality setting can optionally be provided. This maps to Lossy WEBP compression quality.
+				Additionaly, you may trigger near lossless compression with the "near_lossless" flag.
```

Comment:
```suggestion
				Additionally, you may trigger near lossless compression with [param near_lossless].
```
Spelling, using `param` and lastly removing "flag" because to me that sounds more complicated like a simple bool, like maybe implying a bit flag or enum (YMMV on the last though).